### PR TITLE
DEV2-4088: Improve `global.monitoring.enabled` 

### PR DESCRIPTION
**Pull request by SimSimY**
* Setting the flag to true will enable import of grafana dashboards for used services. 
* For some services, just enabling the service monitoring object doesn't allow collection of metrics as they needs to also be exposed by the source service. This PR ensures that the sub-charts properly support the flag.

Overall, assuming Grafana and Prometheus are properly configured, with this flag clients will receive all the dashboards and collect metrics (expose `ServiceMonitor`s and `PodMonitor`s) :

https://github.com/codota/helm-charts/assets/1699509/32d31041-750f-4021-b921-8a731c41f48a

### CODE REVIEW

File: charts/bitnami-postgresql/templates/primary/metrics-svc.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.metrics.enabled }}
```
Comment (yanir-codota) [inline charts/bitnami-postgresql/templates/primary/metrics-svc.yaml:6]: what is metrics svc?

File: charts/tabnine-cloud/README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ A Helm Tabnine Self Hosted Cloud
 | auth.tolerations | list | `[]` |  |
 | backup.backupAllNamespaces | bool | `false` | Should backup all namespaces or just the current one. Defaults to "false". |
 | backup.defaultVolumesToFsBackup | bool | `true` | Should back up the default volumes (non-PVC pod volumes). Defaults to "true". |
-| backup.enabled | bool | `false` | Enable backup for this installation. Disabled by false. Velero must be installed before enabling this flag. |
```
Comment (yanir-codota) [inline charts/tabnine-cloud/README.md:82]: why delete?

File: charts/tabnine-cloud/README.md [hunk 3]
```diff
@@ -152,30 +152,34 @@ A Helm Tabnine Self Hosted Cloud
...
+| global.monitoring.dashboards.namespace | string | `""` | namespace to ta create to the dashboard's configMap, defaults to installation namespace. |
 | global.monitoring.enabled | bool | `false` |  |
 | global.monitoring.labels | object | `{}` |  |
 | global.networkPolicy.enabled | bool | `true` |  |
 | global.tabnine.db.caBase64 | string | `nil` | Use when connecting to the db using SSL (recommended) |
 | global.tabnine.db.certBase64 | string | `nil` | Use when connecting to the db using SSL (recommended) |
 | global.tabnine.db.certPrivateKeyBase64 | string | `nil` | Use when connecting to the db using SSL (recommended) |
 | global.tabnine.db.ip | string | `nil` | Set to allow connection when using networkPolicy.enabled=true |
-| global.tabnine.db.url | string | `nil` | Connection string to postgresDB, in the form: postgres://username:password@host |
+| global.tabnine.db.url | string | `nil` |  |
```
Comment (yanir-codota) [inline charts/tabnine-cloud/README.md:166]: why?

File: charts/tabnine-cloud/README.md
Comment (yanir-codota) [inline charts/tabnine-cloud/README.md:1]: Why is the documentation/description deleted?

**Review by yanir-codota — APPROVED** [review]
LGTM

File: charts/bitnami-postgresql/templates/primary/metrics-svc.yaml (hunk 1 above)
Comment (SimSimY) [reply charts/bitnami-postgresql/templates/primary/metrics-svc.yaml:6]: Thoose are redis internal objects - in this case, this is the k8s svc that is used by the servicemonitor in order to discover relevant pods

**Pull request merged**